data/transport missions.txt

```diff
@@ -3469,3 +3469,114 @@ mission "Inheritance Redirection"
+			choice
+				`	"Sure, I'll help you."`
+					accept
+				`	"Sorry, I don't have time to help you."`
+					decline
+	on visit
+		dialog `You land on <destination>, but the Hauler prototype didn't catch up with you yet.`
+	on complete
+		dialog
+			`You are waiting for the Hauler prototype to land, but it remains in orbit.`
+			`There is no response to your hails by radio. You need to take off and investigate.`
+
+mission "Hauler VI cargo prototype 2"
+	name "Escort Hauler prototype to student party"
+	description "Escort the Hauler prototype to <destination> to deliver cooked fish dishes to a streetfood festival."
+	source "Skymoot"
+	destination "Zug"
+	landing
+	to offer
+		has "Hauler VI cargo prototype 1: done"	
+	npc accompany
+		government "Merchant"
+		personality escort timid
+		ship "Hauler III" "SB-Labs Testament"
+	on enter
+		dialog
+			`As soon as you take off you receive an angry voice message over a short-range inter-ship communications channel, "Why were you trying to contact me over radio! Does everyone need to know that we are flying around here?"`
+			`She adds, "The cooling failed and the sushi nearly cooked, let's go to <destination> to deliver it as fish dishes to a streetfood festival."`
+	on visit
+		dialog `You land on <destination>, but the Hauler prototype didn't catch up with you yet.`
+	on complete
+		dialog
+			`This time the Hauler prototype appears to be landing right behind you, but the ship flies away at full thrust just before touching the landing pad.`
+			`You receive a short text-message over the short-range inter-ship communiation channel, "The sushi overcooked! We cannot be seen near the festival or near the shipyard with this overcooked sushi!"`
+			`The new destination is a floating factory on Mere.`
+
+mission "Hauler VI cargo prototype 3"
+	name "Escort Hauler prototype to factory"
+	description "Escort the Hauler prototype to <destination> to deliver overcooked fish to a floating catfood factory."
+	source "Zug"
+	destination "Mere"
+	landing
+	to offer
+		has "Hauler VI cargo prototype 2: done"
+	npc accompany
+		government "Merchant"
+		personality escort timid
+		ship "Hauler III" "SB-Labs Testament"
+	on visit
+		dialog `You land on <destination>, but the Hauler prototype didn't catch up with you yet.`
+	on complete
+		dialog
+			`The Hauler prototype enters the planets atmosphere at high speed and lands way ahead of you near a catfood factory on one of the floating artifical islands.`
+			`Your infrared sensors show that the ship is emitting a lot of heat, but it's not comming from the heavily used thrusters, rather from the cargo-hold.`
```

Comment:
```suggestion
			`Your infrared sensors show that the ship is emitting a lot of heat, but it's not coming from the thrusters, but rather from the cargo hold.`
```

Is this implying that burned sushi could emit more heat than a ship's thrusters?

Reply:
Not the sushi itself, but the overheating refrigeration/cooling system.

Reply:
Then you need to make that clear. :-)